stream-chat-android-offline/src/main/java/io/getstream/chat/android/livedata/service/sync/PushMessageSyncHandler.kt

```diff
@@ -144,30 +128,6 @@ public class PushMessageSyncHandler(private val service: Service) {
         }
     }
 
```

Comment:
We can't initilizae `client` or `domain` here, because we are unaware what parameters to pass. For example, in our sample app we pass `SampleNotificationHandler`. And that is one of the reasons why push notifications don't work when app process is killed.